include/drives.h

```diff
@@ -75,13 +76,15 @@ class localDrive : public DOS_Drive {
 	virtual bool isRemovable(void);
 	virtual Bits UnMount(void);
 	const char* getBasedir() {return basedir;};
+	virtual bool isNewWriteProtectedFile(const std::string& filename);
```

Comment:
Maybe `virtual bool isNewWriteProtectedFile(const std::string& filename) const;` ?

Reply:
The function updates the member `set<>` if `filename` isn't found though.